### PR TITLE
feat(hyd): control surfaces position impacts flight model

### PR DESCRIPTION
**Pull request by crocket63**
Fixes #[issue_no]

## Summary of Changes

This PR will serve as a sandbox to test linking actual hydraulic surfaces to flight model.

- [x] Roll impact from ailerons
- [x] Roll impact from spoilers
- [x] Roll impact from elevator assymetry
- [x] Pitch impact from elevator  
- [x] Yaw impact from rudder

## Additional context

Discord username (if different from GitHub):

## Testing instructions

Check that no impact is noticeable from before in nominal conditions. Plane reaction in pitch/roll/yaw should be the same as before.
Check that degraded hydraulics conditions causes plane to react overall slower (the effective realism of rates of movement in degraded hydraulics will be fine tuned later)

Secondary quest: try this using YourControls while not beeing the active pilot.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by BlueberryKing — DISMISSED** [review]

File: src/fbw/src/FlyByWireInterface.cpp [hunk 1]
```diff
@@ -1547,13 +1546,32 @@ bool FlyByWireInterface::updateFlyByWire(double sampleTime) {
...
+  idElevatorPosition->set(flyByWireOutput.output.eta_pos);
+
+  // set rudder demand
+  idRudderPosition->set(flyByWireOutput.output.zeta_pos);
+
+  // use direct signals from input when requested
+  if (idDevelopmentUseDirectLaw->get() == 1) {
+    idElevatorPosition->set(-1.0 * simInput.inputs[0]);
+    idRudderPosition->set(-1.0 * simInput.inputs[2]);
+  }
```
[the remark this answers is not in the document]
Comment (crocket63) [reply src/fbw/src/FlyByWireInterface.cpp:1573]: Agreed, but maybe the advantage of this is that it made the commit of adding/removing this functionality separate from everything else. 
Not a big deal to add if / else though

File: src/systems/a320_systems_wasm/src/ailerons.rs [hunk 2]
```diff
@@ -34,5 +39,58 @@ pub(super) fn ailerons(builder: &mut MsfsAspectBuilder) -> Result<(), Box<dyn Er
         Variable::named("HYD_AILERON_RIGHT_DEFLECTION"),
     );
 
+    // AILERON POSITION FEEDBACK TO SIM
+    // Here we separate ailerons from spoiler to build a unique roll torque
+    // Assymetry of elevator adds a part in roll torque
```
[the remark this answers is not in the document]
Comment (crocket63) [reply src/systems/a320_systems_wasm/src/ailerons.rs:44]: God even in French that's a mistake lol

File: src/fbw/src/FlyByWireInterface.cpp [hunk 3]
```diff
@@ -245,6 +248,7 @@ void FlyByWireInterface::setupLocalVariables() {
   idPerformanceWarningActive = make_unique<LocalVariable>("A32NX_PERFORMANCE_WARNING_ACTIVE");
 
   // register L variable for external override
+  idTrackingMode = make_unique<LocalVariable>("A32NX_FLIGHT_CONTROLS_TRACKING_MODE");
   idExternalOverride = make_unique<LocalVariable>("A32NX_EXTERNAL_OVERRIDE");
```
[the remark this answers is not in the document]
Comment (crocket63) [reply src/fbw/src/FlyByWireInterface.cpp:252]: No idea that's Aguther territory :D 
I think this is in fact to use YourControls, where it sets the flight controls from an external source

File: src/systems/a320_systems_wasm/src/elevators.rs [hunk 4]
```diff
@@ -1,27 +1,127 @@
...
 
+    // ELEVATOR POSITION FEEDBACK TO SIM
+    builder.map_many(
+        ExecuteOn::PostTick,
+        vec![
+            Variable::named("HYD_ELEVATOR_LEFT_DEFLECTION"),
+            Variable::named("HYD_ELEVATOR_RIGHT_DEFLECTION"),
+            Variable::named("ELEVATOR_DEFLECTION_DEMAND"),
+            Variable::aircraft("ELEVATOR DEFLECTION PCT", "Percent", 0),
+        ],
```
[the remark this answers is not in the document]
Comment (crocket63) [reply src/systems/a320_systems_wasm/src/elevators.rs:60]: Totally right, forgot to remove those after removing the print statement they were there for.....

**Review by 2hwk — APPROVED** [review]

**Pull request merged**